website/README.md

```diff
@@ -3,13 +3,29 @@
 This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
 
 ### Installation
+Use `npm` or `yarn` to setup and test website on local environment.
```

Comment:
There are many ways to run and build this package (i personally build it with `bun`). Instead of linking all the possible tools, shall we just add a link, stating "If you're using other tools than npm, you can refer to the docusaurs documentation for build and installation https://docusaurus.io/docs/installation#build"